cpp/src/barretenberg/dsl/acir_format/recursion_constraint.cpp

```diff
@@ -113,12 +122,14 @@ void create_recursion_constraints(Composer& composer, const RecursionConstraint&
 
     // Assign the output aggregation object to the proof public inputs (16 field elements representing two
     // G1 points)
-    result.add_proof_outputs_as_public_inputs();
+    // result.add_proof_outputs_as_public_inputs();
```

Comment:
```suggestion
```

Reply:
Do you think I should remove the comment above as well? It may be confusing to leave it otherwise?